instrumentation/spring/spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -1,5 +1,6 @@
 plugins {
   id("otel.library-instrumentation")
+  id("otel.japicmp-conventions")
```

Comment:
I don't think this will do anything without also marking the module as stable (and then the api diff files should get generated)

Reply:
should I make it stable or remove the line?

Reply:
Since this PR ended up doing other things I'd suggest removing this line and updating the PR title

Reply:
Then we can merge it and you can send a new PR to mark stable with the apidiff that we can review

Reply:
done